kubejs/client_scripts/base/jei_descriptions.js

```diff
@@ -27,25 +27,33 @@ JEIEvents.information((event) => {
             text: ['Obtained by Right-Clicking a Glass Bottle in the air between Y Levels 192 and 196.']
         },
         {
-            items: [Item.of('naturesaura:aura_bottle', { stored_type: 'naturesaura:overworld' })],
+            items: ['naturesaura:aura_bottle'],
             text: [
-                'Obtained by Right-Clicking a Bottle and Cork in the air in the Overworld, Twilight Forest, or the Everdawn. This action removes Aura from the area.',
+                'Obtained by Right-Clicking a Bottle and Cork in the air in the appropriate dimension. This action removes Aura from the area.',
+                ' ',
+                `Bottled Sunlight: The Overworld, Twilight Forest, or the Everdawn`,
+                `Bottled Ghosts: Nether or Everbright`,
+                `Bottled Darkness: The End or The Bumblezone`,
                 ' ',
                 `Can be automated using a Dispenser.`
             ]
         },
         {
-            items: [Item.of('naturesaura:aura_bottle', { stored_type: 'naturesaura:end' })],
+            items: ['naturesaura:bottle_two_the_rebottling'],
```

Comment:
🤣

Reply:
Yeah, not really sure what happened here. The nbt types were working to show descriptions in JEI before (I think). This isn't ideal, but it's better than nothing.